app/models/host.rb

```diff
@@ -1304,7 +1308,7 @@ def set_custom_field(attribute, value)
 
   def quickStats
     return @qs if @qs
-    return {} unless is_vmware?
+    return {} unless supports_quick_stats?
```

Comment:
awesome, these are the nice one line changes that make me happy :)

cc @blomquisg @agrare